src/main/scala/viper/silver/plugin/PluginTemplate.scala

```diff
@@ -53,13 +54,23 @@ class PluginTemplate extends SilverPlugin {
       */
     override def beforeVerify(input: Program) : Program = ???
 
+    /** Called after the verification of an entity. Error transformation should happen here.
```

Comment:
Maybe add another line to explain why we have entity verification results AND overall verification results?

Reply:
Good idea, thanks!